packages/shorebird_cli/test/src/commands/init_command_test.dart

```diff
@@ -883,6 +883,31 @@ flutter:
       );
     });
 
+    test('logs success message on completion', () async {
+      await runWithOverrides(command.run);
+      verify(
+        () => logger.info(
+          any(
+            that: stringContainsInOrder(
+              [
+                lightGreen.wrap('🐦 Shorebird initialized successfully!')!,
+                '✅ A shorebird app has been created.',
+                '✅ A "shorebird.yaml" has been created.',
+                '''✅ The "pubspec.yaml" has been updated to include "shorebird.yaml" as an asset.''',
+                '''shorebird.yaml does not contain any sensitive information and should be checked into version control.''',
```

Comment:
Should we emphasize "public version control"?

Reply:
I don't know that users will always want to check their app into public version control. Most companies are closed-source by default.